docs/getting-started.md

```diff
@@ -0,0 +1,64 @@
+| OSI Model           | ([link](https://en.wikipedia.org/wiki/OSI_model))                                                                           | N/A                                                          | High level overview of how information gets transmitted between two computer systems                                                                                                         |
+| HTTP Protocol       | ([link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview))<br/>([reference](https://tools.ietf.org/html/rfc7230)) | N/A<br/>Yes                                                  | Protocol used for communication between two computer systems. Essential for modern software development                                                                                      |
+| REST API            | ([link](https://en.wikipedia.org/wiki/Representational_state_transfer))                                                     | N/A                                                          | An approach simplifying use of HTTP requests for client-server communication and actions                                                                                                     |
+| Pro Git book        | ([link](https://git-scm.com/book))                                                                                          | Read sections 1-3, and can skim through the rest             | Git is extremely useful and powerful source code management tool. With our approach 'Everything as Code', its essential to know how to use it                                                |
+| JavaScript tutorial | ([link](https://nodejs.dev/learn))                                                                                          | N/A                                                          | Any JavaScript tutorial of your choice. We recommend a couple of books/courses, focused on NodeJS (vs browser)                                                                               |
+| NodeJS API          | ([link](https://nodejs.org/docs/latest/api/))                                                                               | Yes                                                          | It is impossible to write software without knowing standard capabilities of the chosen platform. Skim through NodeJS API just to know what functionality is available for you out of the box |
+| Markdown language   | ([link](https://guides.github.com/features/mastering-markdown/))                                                            | Yes                                                          | Markup language used by nearly all code collaboration tools, in comments, chat messages, documents etc.                                                                                      |
```

Comment:
Note that I've added markdown to the list - the language this comment, and pull request is written in